openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -21,30 +21,32 @@ def process(self, context):
 
         self.log.debug("Comment is `{}`".format(publish_comment))
 
-        # Get note status, by default uses the task status for the note
-        # if it is not specified in the configuration
-        note_status = context.data["kitsu_task"]["task_status_id"]
-        if self.set_status_note:
-            kitsu_status = gazu.task.get_task_status_by_short_name(
-                self.note_status_shortname
-            )
-            if kitsu_status:
-                note_status = kitsu_status
-                self.log.info("Note Kitsu status: {}".format(note_status))
-            else:
-                self.log.info(
-                    "Cannot find {} status. The status will not be "
-                    "changed!".format(self.note_status_shortname)
-                )
+        for instance in context:
+
+            # Get note status, by default uses the task status for the note
+            # if it is not specified in the configuration
+            note_status = instance.data["kitsu_task"]["task_status"]["id"]
```

Comment:
Task may not be filled so it should allow to skip instances without tasks
```suggestion
        for instance in context:
            kitsu_task = instance.data.get("kitsu_task")
            if kitsu_task is None:
                continue

            # Get note status, by default uses the task status for the note
            # if it is not specified in the configuration
            note_status = kitsu_task["task_status"]["id"]
```

Reply:
True true. Fixed in latest push